From 1658d5528020ccd0739d91cd957d0029a4a7d971 Mon Sep 17 00:00:00 2001
From: Andy Duan <fugang.duan@nxp.com>
Date: Tue, 26 Jul 2016 17:17:25 +0800
Subject: [PATCH 1114/5242] MLK-13008 dts: imx6ull-evk: add Murata BT/WiFi
 module support

commit  d1b44b8a19ee71c922c6f89c50252bd218ee2a15 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add Murata Type ZP (BCM4339) module support on i.MX6ULL platforms:
- i.MX6ULL EVK (SD1 slot) + Murata adapter V2.0

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/Makefile                     |    3 ++-
 arch/arm/boot/dts/imx6ull-14x14-evk-btwifi.dts |   10 ++++++++++
 2 files changed, 12 insertions(+), 1 deletion(-)
 create mode 100644 arch/arm/boot/dts/imx6ull-14x14-evk-btwifi.dts

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index a284b94..e99446c 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -618,7 +618,8 @@ dtb-$(CONFIG_SOC_IMX6ULL) += \
 	imx6ull-14x14-ddr3-arm2-wm8958.dtb \
 	imx6ull-14x14-evk.dtb \
 	imx6ull-colibri-eval-v3.dtb \
-	imx6ull-colibri-wifi-eval-v3.dtb
+	imx6ull-colibri-wifi-eval-v3.dtb \
+	imx6ull-14x14-evk-btwifi.dtb
 dtb-$(CONFIG_SOC_IMX7D) += \
 	imx7d-cl-som-imx7.dtb \
 	imx7d-colibri-emmc-eval-v3.dtb \
diff --git a/arch/arm/boot/dts/imx6ull-14x14-evk-btwifi.dts b/arch/arm/boot/dts/imx6ull-14x14-evk-btwifi.dts
new file mode 100644
index 0000000..8a0a85d
--- /dev/null
+++ b/arch/arm/boot/dts/imx6ull-14x14-evk-btwifi.dts
@@ -0,0 +1,10 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6ull-14x14-evk.dts"
+#include "imx6ul-evk-btwifi.dtsi"
-- 
1.7.9.5

